### PR TITLE
chore(environments): add amplify/hooks to environments directory .gitignore

### PR DESCRIPTION
**Pull request by calebpollman**
#### Description of changes
Add `amplify/hooks` to environments directory _.gitignore_

#### Checklist

- [x] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by joebuono — APPROVED** [review]

File: environments/.gitignore [hunk 1]
```diff
@@ -6,11 +6,12 @@
 # Auto-generated by amplify, but with `**` added to cover all environments
 **/amplify/\#current-cloud-backend
 **/amplify/.config/local-*
-**/amplify/logs
-**/amplify/mock-data
 **/amplify/backend/amplify-meta.json
 **/amplify/backend/awscloudformation
 **/amplify/backend/.temp
+**/amplify/hooks
```
Comment (zchenwei) [inline environments/.gitignore:12]: QQ: what is under the hooks folder? Shouldn't content in this file be auto-generated?

Comment (calebpollman) [reply environments/.gitignore:12]: There is a `README` that escapes our current `.gitignore` patterns

File: environments/.gitignore [hunk 2]
```diff
@@ -6,11 +6,12 @@
 # Auto-generated by amplify, but with `**` added to cover all environments
 **/amplify/\#current-cloud-backend
 **/amplify/.config/local-*
-**/amplify/logs
-**/amplify/mock-data
```
[the remark this answers is not in the document]
Comment (calebpollman) [reply environments/.gitignore:10]: Not sure, i did not do an audit

File: environments/.gitignore (hunk 1 above)
Comment (zchenwei) [reply environments/.gitignore:12]: Ouch..

**Review by zchenwei — APPROVED** [review]

**Review by wlee221 — APPROVED** [review]

**Pull request merged**